extra/mender-client-docker-addons/Dockerfile

```diff
@@ -38,12 +42,12 @@ RUN mender-artifact write bootstrap-artifact \
         --output-path /bootstrap.mender
 
 WORKDIR /src/mender-connect
-RUN make prefix=/mender-install install
+RUN DESTDIR=/mender-install make prefix=/usr install
```

Comment:
I'm guessing you want this reverted to the old setup here?

But in a sense, it bothers me that we don't do this properly.

Would it be off if I fixed this in the given repos?